zxlive/mainwindow.py

```diff
@@ -66,16 +65,21 @@ def __init__(self) -> None:
         def Button_Fuse_Clicked():
             g, vs = self.get_elements()
             if vs == []:
+                self.graph_view.set_graph(g)
                 return
-            self.backup()
+
+            new_g = copy.deepcopy(g)
```

Comment:
Pyzx has a `BaseGraph.copy()` function. Do you maybe want to use that here instead? That copy function does change the labels of the vertices, so that might not be desirable.

Reply:
Changing labels sounds troublesome, since the VItems carry the original labels of the vertices.